tap_ga4/reports.py

```diff
@@ -448,6 +460,17 @@
             "itemBrand",
         ],
     },
+    {
+        "name": "in_app_purchases",
+        "metrics": [
+            "itemsPurchased",
+            "itemRevenue",
+        ],
+        "dimensions": [
+            "date",
+            "itemId",
+        ],
+    },
```

Comment:
Am I correct in assuming this list was built from the GA4 UI?

Reply:
Yes, but I just double checked and the "in_app_purchases" report has an extra metric in the GA4 UI, which is not available in the API. I'll check if we want to keep this as is.

Reply:
Checked with product, we ok with omitting the last metric because the users would be able to calculate it themselves, see thread: https://talend.slack.com/archives/C03THJYD0U9/p1670426826856869